### ISSUE TITLE
Some issues on continue training

### ISSUE THREAD

**Issue by yttdebaba2** [Microsoft/CNTK#1566]
I want to train a model with the previously trained model.
So I move `wordembedding.17` and `wordembedding.17.ckp` from folder `work` to folder `work2`, and set
`modelPath = work2/wordembedding`
But it seems that the model is trained from the very beginning, since the loss is much higher compared to the loss with model `wordembedding.17`.
So what else should I do to continue my training using the pre-trained model.

**Comment by eldakms** [comment]
Could you please share your python script so that we can reproduce/check how you save/restore from the checkpoint. Thanks!

**Comment by yttdebaba2** [comment]
In fact I used BrainScript as below:
`WorkDir = work2`

`makeMode = false`
`modelPath = $WorkDir$/wordembedding`
`#keepCheckPointFiles = True`

`command = Train`
`Train = [`
`...`
`]`

**Comment by eldakms** [comment]
I see you have a makeMode=false, this means always start from scratch. Could you please set it to true and retry? Thanks!

**Comment by yttdebaba2** [comment]
I success by setting makeMode=true. Thank you very much!
